src/commonTest/kotlin/uk/gov/hmrc/CalculatorTest.kt

```diff
@@ -71,4 +76,103 @@ class CalculatorTest {
             Calculator(regularPayment = 51).run()
         }
     }
+
+    @Test
+    fun `return month breakdown if they have only saved in the first 2 months`() {
```

Comment:
The formatting is a little inconsistent in these tests below.. not sure if we've discussed following any code style conventions yet for multiplatform..